src/runners/run-ci.sh

```diff
@@ -24,7 +24,7 @@ function get_secret_values() {
 
     if [[ ! -z "${account_number}" && "${OVERRIDE_ROLE}" == "true" ]]; then
         # Set for local running
-        export AWS_ROLE_ARN="${arn_value}:${aws_value}:${service}::${account_number}:role/administrator"
+        export AWS_ROLE_ARN="${arn_value}:${aws_value}:${service}::${account_number}:role/RestrictedAdmin"
```

Comment:
Remove this change please. Ill fix this properly in another commit, thanks

Reply:
Sure will remove that change, Thanks Rory